model/service.go

```diff
@@ -44,15 +45,28 @@ type Endpoint struct {
 	// Address of the endpoint, typically an IP address
 	Address string `json:"ip_address,omitempty"`
 	// Port on the host address
-	Port int `json:"port,omitempty"`
+	Port int `json:"port"`
+	// Name of the port classifies ports for a single service
+	Name string `json:"name,omitempty"`
```

Comment:
There could be multiple HTTP ports for a service. We have to further quality each port by a name to distinguish between them. I have not used this field yet, basically assuming there is only one port for a service.